frontend/app/models/census-tract.js

```diff
@@ -0,0 +1,35 @@
+import DS from 'ember-data';
+const { Model } = DS;
+import { attr } from '@ember-decorators/data';
+import { computed } from '@ember-decorators/object';
+
+/**
+ * A censusTract record represents a single transportation census data variable for a given geoid.
+ * Four individual census tract objects represent the full data for a single geoid:
+ * - population
+ * - trans_total
+ * - trans_auto_total
+ * - trans_public_total
+ * where trans_* variables are modal split data.
+ */
+export default class CensusTractModel extends Model {
+  @attr('string') geoid;
+  @attr('string') variable;
+  @attr('number') value;
+  @attr('number') moe;
+
+  /**
+   * Returns the human-readable "mode" string for a given census-tract variable
+   */
+  @computed('variable')
+  get mode() {
+    const variable = this.get('variable');
+    return VARIABLE_MODE_LOOKUP[variable] || 'Unknown';
+  }
```

Comment:
Gorgeous.
non-blocking (bikeshedding?): Only tiny possible issue: the human-readable version of an 'Unknown' is represented in multiple places across the app (See https://github.com/NYCPlanning/ceqr-app/pull/344/files#diff-d323bc5d7c3fca3306820fb54a4a72dbR24). Maybe the lookup could be wrapped in a "crosswalk" function that uses `CASE WHEN`, having a default?

Reply:
Yes, but that other place is checking that there is a value for the population variable, not looking up  a human-readable translation of a variable name. Adding a crosswalk here does not get rid of the other 'Unknown' check.